src/App.js

```diff
@@ -0,0 +1,19 @@
+import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Header from './components/Header';
+import Categories from './components/Categories';
+import BookList from './components/BookList';
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <Routes>
+        <Route path="BookList" element={<BookList />} />
```

Comment:
- [ ] Kindly make your route to be `<Route exact path="/">` so that  not show a blank page when visiting the site for the first time, the BookList component should be the default page.